rfc/expr-context.md

```diff
@@ -0,0 +1,142 @@
+# Expression Context
+
+Cosmo Router provides a context object to an expr-lang VM that can be used to access information about the request, response, and other relevant information. The context object is available at all times during the request's lifecycle.
+
+## request
+
+The client request object is a read-only object that provides information about the incoming request. It is available at all times during the request's lifecycle.
+
+```
+request.id
+request.header
+request.method
+request.body.query
+request.body.operation_name
```

Comment:
We should stick with camelCase as we do everywhere else.

Reply:
For simplicity, I suggest to be `CamelCase` (first letter uppercase) because it works that way with Go exported methods. Otherwise, we would need to construct an any map.